nautobot/extras/urls.py

```diff
@@ -200,6 +200,19 @@
         views.JobJobResultView.as_view(),
         name="job_jobresult",
     ),
+    path("jobs/schedules/", views.ScheduledJobListView.as_view(), name="scheduled_jobs_list"),
```

Comment:
API has a separate `/scheduled-jobs/` urlpattern for these - would be preferable to be consistent between API and UI.